php/ClassInfo/ComposerClassmapFileRepository.php

```diff
@@ -104,13 +100,18 @@ public function find($path_pattern, ClassFilter $filter = null)
      */
     private function isValid($classpath)
     {
-        $filePath = $this->classLoader->findFile($classpath);
```

Comment:
Dependency of composer's `ClassLoader` is proper within `ComposerClassmapFileRepository`, but I don't assume that the implementation is ultimate and only for the project (as I described above). It is transitional and uses information that were easy to get (classmap and Reflection). More important are interfaces that allows replace such poor implementations by something better or even allows to use more than one implementation if needed.